project_task_default_stage/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     'name': 'Project Task Default Stage',
     'summary': 'Recovery default task stage projects from v8',
-    'version': '10.0.1.0.0',
+    'version': '10.0.2.0.0',
```

Comment:
Thought the same since it's a small change though it matched better the middle version that said it should require module update. Also if someone should do a position after the same element it would change the design slightly.

But if you think it should increment the last number I can make the change, your call.